buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -70,10 +76,14 @@ pipeline {
 						if (env.BUILD_LIST == 'openjdk_regression' ||  env.BUILD_LIST == 'thirdparty_containers') {
 							DIAGNOSTICLEVEL ='noDetails'
 						}
-					}
-					withEnv(["DIAGNOSTICLEVEL=${DIAGNOSTICLEVEL}"]) {
-						sh 'printenv'
-                		sh "$OPENJDK_TEST/maketest.sh $OPENJDK_TEST _${params.TARGET}"
+					
+						withEnv(["DIAGNOSTICLEVEL=${DIAGNOSTICLEVEL}"]) {
+							sh 'printenv'
+							int iterations = "${ITERATIONS}".toInteger()
```

Comment:
Not sure if the following script  needs  to be wrapped in script {}.

Reply:
A previous thing I tried did need it, I will try now without it to see if it works without.  Thanks!

Reply:
Need to keep the .toInteger() inside the block